PR for SRVLOGIC-189: Added the installtion and uninstallation assemblies for Openshift Serverless Logic docs (#76488)
openshift/openshift-docs: install/install-serverless-logic-operator.adoc

:_content-type: ASSEMBLY
include::_attributes/common-attributes.adoc[]
[id="install-serverless-logic-operator"]

> = Installing the {ServerlessLogicOperatorName}

Reviewer: ```suggestion
= Installing the {ServerlessLogicProductName}
```

:context: install-serverless-logic-operator

toc::[]

> Installing the {ServerlessLogicOperatorName} enables you to install and use link:https://serverlessworkflow.io[Serverless Workflow] on a {ocp-product-title} cluster. The {ServerlessLogicOperatorName} manages Kubernetes custom resource definitions (CRDs) to configure and deploy the workflows.

Reviewer: ```suggestion
Installing the {ServerlessLogicProductName} enables you to install and use link:https://serverlessworkflow.io[Serverless Workflow] on a {ocp-product-title} cluster. The {ServerlessLogicOperatorName} manages Kubernetes custom resource definitions (CRDs) to configure and deploy the workflows.
```

> Installing the {ServerlessLogicOperatorName} enables you to install and use Knative components on a {ocp-product-title} cluster. The {ServerlessLogicOperatorName} manages Knative custom resource definitions (CRDs) for your cluster and enables you to configure them without directly modifying individual config maps for each component.

Reviewer 2: ```suggestion
Installing the {ServerlessLogicOperatorName} enables you to install and use serverless workflows on a {ocp-product-title} cluster. The {ServerlessLogicOperatorName} manages Kubernetes custom resource definitions (CRDs) to configure and deploy the workflows.
```

Reviewer 2: wdyt? @domhanak

Reviewer 3: Please add a link to the `https://serverlessworkflow.io` in `serverless workflows`.

Author: Added

:_content-type: ASSEMBLY
include::_attributes/common-attributes.adoc[]
[id="install-serverless-logic-operator"]
= Installing the {ServerlessLogicOperatorName}
:context: install-serverless-logic-operator

toc::[]

> Installing the {ServerlessLogicOperatorName} enables you to install and use link:https://serverlessworkflow.io[serverless workflow] on a {ocp-product-title} cluster. The {ServerlessLogicOperatorName} manages Kubernetes custom resource definitions (CRDs) to configure and deploy the workflows.

Reviewer 4: It appears that they capitalize Serverless Workflow upstream

```suggestion
Installing the {ServerlessLogicOperatorName} enables you to install and use link:https://serverlessworkflow.io[Serverless Workflow] on a {ocp-product-title} cluster. The {ServerlessLogicOperatorName} manages Kubernetes custom resource definitions (CRDs) to configure and deploy the workflows.
```
